bin/helm-deploy

```diff
@@ -21,11 +31,18 @@ do
   CHART_PATH=${HELM_CHARTS[$index]}
   CHART_RELEASE_NAME=${HELM_RELEASE_NAMES[$index]:-$CHARTS[$index]}
   CHART_VALUES=${HELM_VALUES[$index]}
+
+  # if there is more than one values files listed
+  # then pass all values file to helm upgrade command
+  if [[ "${CHART_VALUES}" = *","* ]]; then
+    CHART_VALUES=$(format_multiple_values_files "${CHART_VALUES}")
+  else:
```

Comment:
Does `else:` work in bash? I'm getting an error.

Reply:
good catch, removed colon